Make sure mv does not prompt while overwriting the target file.
See also https://github.com/git/git/commit/861dc19ba852cf668b40d0ef601b44c7c03853a1

--- git-2.45.1/t/lib-chunk.sh.orig
+++ git-2.45.1/t/lib-chunk.sh
@@ -13,5 +13,5 @@
 	fn=$1; shift
 	perl "$TEST_DIRECTORY"/lib-chunk/corrupt-chunk-file.pl \
 		"$@" <"$fn" >"$fn.tmp" &&
-	mv "$fn.tmp" "$fn"
+	mv -f "$fn.tmp" "$fn"
 }
